GraphAlgorithms/Commands/SizeCommand.cs

```diff
@@ -0,0 +1,28 @@
+﻿using System.CommandLine;
+
+namespace GraphAlgorithms.Commands;
+
+public static class SizeCommand
+{
+    public static Command Create()
+    {
+        var fileOption = OptionsFactory.CreateFileOption();
+        var indexOption = OptionsFactory.CreateIndexOption();
+
+        var command = new Command("size", "Calculate size of a graph.")
+        {
+            fileOption,
+            indexOption
+        };
+        command.SetHandler(Run, fileOption, indexOption);
+
+        return command;
+    }
+
+    private static void Run(string path, int index)
+    {
+        var graph = Utils.GetGraph(path, index);
+
+        Console.WriteLine($"Graph size: ({graph.Size}, {graph.GetNumberOfEdges()})");
```

Comment:
this is not regarding the code, but just a side note: we probably should write in the document that we define size like that and we order them "lexicographically" (or whatever the clever word for that was)